### ISSUE TITLE
a11y: Navigating expanded powerSelect with the keyboard doesn't cause the selected item to be spoken by screen reader.

### ISSUE THREAD

**Issue by MarcoZehe** [cibernox/ember-power-select#1369]
Steps to reproduce:
1. With NVDA and Firefox, in the sample app, navigate to the PowerSelect.
2. Press Space to open.
3. Once expanded, press up and down arrows.

Expected: NVDA would read the selected item.
Actual: Nothing is spoken. Or NVDA doesn't even switch to focus mode and doesn't recognize that the dropdown opened.

With VoiceOver on the Mac, seeing the same thing. Up and Down don't speak the new value once the dropdown is open.

**Comment by MarcoZehe** [comment]
To be clearer, this component needs to implement the full [combobox design pattern](https://www.w3.org/TR/wai-aria-practices-1.2/#combobox) from the WAI-ARIA authoring practices. It currently only implements pieces of it and is therefore not fully accessible to users of assistive technologies.

CC @mydea because you previously showed an interest in a11y of this component, and @jayjayjpg because we talked about accessibility in the Ember.js eco system before.

**Comment by MelSumner** [comment]
@cibernox there are a couple of blocking a11y issues that prevent us from using this addon at work- this issue is one of them. Would you be open a chat about how we can help these get unblocked?

**Comment by arthur5005** [comment]
a11y should be a top priority, if not, _the_ priority for any UI component anywhere. Happy to help with some general guidance.

**Comment by uhrohraggy** [comment]
Any movement on this?

**Comment by ryanromanov** [comment]
Also hoping to see some movement on this. We're experiencing the same issue as @MarcoZehe with seemingly no way around it other than just implementing our own component instead of using power select.

**Comment by MelSumner** [comment]
I sort of thought this was an issue in general and that we'd fixed it in ember-source, but I also suspect that power-select is perhaps trying to do a little bit too much (technically possible but not supported by a11y tooling). 

We could take up this discussion in Discord in #topic-a11y and see what the path forward might look like for accessible selects.

**Comment by mkszepp** [comment]
There was extended the support for a11y in PR #1690
Please let me know if we can close this issue or if we need additional changes for this
